### PR TITLE
Add test helper for parsing types

### PR DESCRIPTION
**Pull request by alexkwolfe**
The new `parse` function is for testing and would replace the use of leadconduit-field's `buildLeadVars` function.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -2,4 +2,76 @@
...
+requestVars = [
+  { name: 'zip', type: 'postal_code' }
+  { name: 'home_phone', type: 'phone' }
+]
+
+parse = parser(requestVars)
+
+parse(name: 'Alex', zip: '78704', home_phone: '5127891121')
+
+# => { name: 'Alex',
```
Comment (cgrayson) [inline Readme.md]: Sweet! Did you leave out listing `name` as a request var to show that data is left as strings by default? That aspect is a little subtle, maybe mention it explicitly.

File: package.json [hunk 2]
```diff
@@ -2,10 +2,12 @@
...
-  "licenses": [{
-    "type": "Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International",
-    "url": "http://creativecommons.org/licenses/by-nc-nd/4.0/legalcode"
-  }],
+  "licenses": [
+    {
+      "type": "Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International",
+      "url": "http://creativecommons.org/licenses/by-nc-nd/4.0/legalcode"
+    }
+  ],
```
Comment (cgrayson) [inline package.json]: This format is no longer valid; it should be [like this](https://github.com/activeprospect/leadconduit-integration-twilio/blob/master/package.json#L6).

**Comment by cgrayson** [comment]
L :+1: TM

File: Readme.md (hunk 1 above)
Comment (alexkwolfe) [inline Readme.md]: Done.

**Pull request merged**